compose/assets/db/dump-voms_vo_0.sql

```diff
@@ -331,7 +331,11 @@ CREATE TABLE `certificate` (
 
 LOCK TABLES `certificate` WRITE;
 /*!40000 ALTER TABLE `certificate` DISABLE KEYS */;
-INSERT INTO `certificate` VALUES (1,'2022-12-29 15:55:35','/C=IT/O=IGI/CN=test0','\0',NULL,NULL,1,1),(2,'2022-12-29 15:55:35','/C=IT/O=IGI/CN=test1','\0',NULL,NULL,1,2),(3,'2022-12-29 15:55:35','/C=IT/O=IGI/CN=(Parenthesis)','\0',NULL,NULL,1,3);
+INSERT INTO `certificate` VALUES 
+  (1,'2022-12-29 15:55:35','/C=IT/O=IGI/CN=test0','\0',NULL,NULL,1,1),
+  (2,'2022-12-29 15:55:35','/C=IT/O=IGI/CN=test1','\0',NULL,NULL,1,2),
+  (3,'2022-12-29 15:55:35','/C=IT/O=IGI/CN=(Parenthesis)','\0',NULL,NULL,1,3),
```

Comment:
Is there a specific need to have a subject with `(Parenthesis)` or is it just because we don't have other test certificates?

Reply:
I don't see it used in the tests indeed, this comes from a db dump of VOMS Admin. I guess we can remove it as to my understanding we are using just test0, test1 and test2 certificates.